src/main/java/uk/gov/companieshouse/acsp/manage/users/controller/AcspMembershipController.java

```diff
@@ -115,6 +124,13 @@ public ResponseEntity<Void> updateAcspMembershipForAcspAndId( final String xRequ
 
         LOG.infoContext( xRequestId, String.format( "Successfully updated Acsp Membership %s", membershipId ), null );
 
+        if ( isOAuth2Request() && Objects.nonNull( userRole ) ){
+            final var requestingUserDisplayName = Optional.ofNullable( requestingUser.getDisplayName() ).orElse( requestingUser.getEmail() );
+            final var targetUser = usersService.fetchUserDetails( membershipIdAssociation.getUserId() );
+            final var targetAcsp = acspProfileService.fetchAcspProfile( membershipIdAssociation.getAcspNumber() );
```

Comment:
also does session have acsp name?

Reply:
The PATCH endpoint takes the `membershipId` as input and retrieves the membership from `acsp_members.acsp_members`. The returned membership contained almost everything that was required e.g. acspNumber - we didn't need the acspName or acspStatus, so never had to fetch the AcspProfile before.

The update method returns void, and even if we were to change it to return the result of the update, it wouldn't contain the acspName, because it would only return the dao.

The session has the acspNumber, but not its name.